frontend/beCompliant/src/hooks/backendUrl.ts

```diff
@@ -0,0 +1,16 @@
+const useBackendUrl = (path?: string) => {
+  const backendUrl = import.meta.env.VITE_BACKEND_URL;
+  const isProd = import.meta.env.PROD;
+
+  if (isProd) {
+    if (backendUrl) {
+      return [backendUrl, path].join('');
+    } else {
+      console.error('Backend URL not found :(');
+    }
+  }
+
+  return ['http://localhost:8080', path].join('');
```

Comment:
```suggestion
  return `http://localhost:8080${path}`
```

This is just personal preference from my side

Reply:
+1 keegt med inline string concat

Reply:
Eller er det for å unngå '' på concat?

Reply:
For og undgå join bare generelt, Min suggestion sender urlen som ren string